src/applications/vaos/new-appointment/components/ReviewPage/index.jsx

```diff
@@ -76,50 +85,52 @@ export default function ReviewPage() {
         </LoadingButton>
       </div>
       {submitStatus === FETCH_STATUS.failed && (
-        <AlertBox
-          status="error"
-          headline="We couldn’t schedule this appointment"
-          content={
-            <>
-              {submitStatusVaos400 ? (
-                <p>
-                  We’re sorry. Something went wrong when we tried to submit your{' '}
-                  {isDirectSchedule ? 'appointment' : 'request'}. You’ll need to
-                  call your local VA medical center to schedule this
-                  appointment.
-                </p>
-              ) : (
+        <div role="alert">
+          <AlertBox
+            status="error"
+            headline="We couldn’t schedule this appointment"
+            content={
+              <>
+                {submitStatusVaos400 ? (
+                  <p>
+                    We’re sorry. Something went wrong when we tried to submit
+                    your {isDirectSchedule ? 'appointment' : 'request'}. You’ll
```

Comment:
Minor, but maybe this ternary could be put into a variable.

Reply:
Yeah, wouldn't hurt. I hadn't changed this code, it's in the diff because the indentation changed.